src/csvcubed/models/sparqlresults.py

```diff
@@ -233,12 +233,12 @@ class MetadataDependenciesResult:
 @dataclass
 class TableSchemaPropertiesResult:
     """
-    Model representing the table schema value url and about url.
+    Model representing the about_url, csv_url and a list of primary key column names.
     """
 
     about_url: str
-    value_url: str
-    table_url: str
+    csv_url: str
+    primary_key_col_names: List[str]
```

Comment:
Could you either change this to a set or put a docstring comment underneath it explaining that the order is not guaranteed?